action.yml

```diff
@@ -18,7 +18,7 @@ inputs:
   version:
     description: Specify runn version
     required: false
-    default: "v0.114.1"
+    default: latest
```

Comment:
> How about setting the default version to the latest version as the runn-action?

The version of runn-action is linked to the version of runn
This version is updated when tagging by running the following job
If runn-action's latest version is specified, runn's latest version is expected to be used if the version is published in conjunction with runn-action.
https://github.com/k2tzumi/runn-action/blob/main/.github/workflows/release.yml

Reply:
Understood 👍 

On the other hand, it seems a waste not to be able to specify `latest`.

Reply:
Also, it looks like you should update the `v0` and `latest` tags.

Reply:
I see
I guess I'll add a process to re-cut the v0 and latest tags in the workflow at the time of release.

Reply:
Thank you!

Reply:
This PR only allows to specify the `latest` version.

Reply:
Issue registered
https://github.com/k2tzumi/runn-action/issues/42
https://github.com/k2tzumi/runn-action/issues/43